build/debs/build.go

```diff
@@ -23,13 +23,12 @@ const (
 	DistributionUnstable DistributionType = "unstable"
 	DistributionTesting  DistributionType = "testing"
 
-	cniVersion         = "0.7.5"
-	criToolsVersion    = "1.13.0"
-	pre180kubeadmconf  = "pre-1.8/10-kubeadm.conf"
-	pre1110kubeadmconf = "post-1.8/10-kubeadm.conf"
-	latestkubeadmconf  = "post-1.10/10-kubeadm.conf"
+	cniVersion      = "0.7.5"
+	criToolsVersion = "1.13.0"
```

Comment:
i think we might want to drop the magic build.go is doing in terms of version handling for cni/cri, long term.
it's fighting the deficiency of not having separate spec files or release branches.